pkg/kubelet/kubelet_node_status.go

```diff
@@ -474,6 +475,24 @@ func (kl *Kubelet) recordNodeSchedulableEvent(node *v1.Node) error {
 // TODO(madhusudancs): Simplify the logic for setting node conditions and
 // refactor the node status condition code out to a different file.
 func (kl *Kubelet) setNodeStatus(node *v1.Node) {
+	// TODO(dixudx): remove this in v1.14
```

Comment:
why remove in v1.14？ this may occur in Ever release version upgrade later than 1.13.

Reply:
@zhangxiaoyu-zidif We do not recommend upgrading more than two minor releases at a time (see https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#upgrades).